components/cdc/src/observer.rs

```diff
@@ -106,6 +126,8 @@ impl<E: KvEngine> CmdObserver<E> for CdcObserver {
     ) {
         assert!(!cmd_batches.is_empty());
         fail_point!("before_cdc_flush_apply");
+        // Untrack raw ts here because it's tracked in CausalObserver::pre_propose_query.
+        self.untrack_raw_ts(cmd_batches);
```

Comment:
There is gap between "untrack raw ts" and "on_multi_batch". It is possible that a `on_min_ts` is inserted in the gap.

Reply:
done, schedule `RawUntrackTs` task after `MultiBatch task` or level is not All.